app/pages/ServiceDiscoveryResults/ServiceDiscoveryResults.tsx

```diff
@@ -172,10 +176,12 @@ const InnerServiceDiscoveryResults = ({
           onSearchStateChange={onSearchStateChange}
         >
           {category.disableGeoLocation ? (
-            <Configure filters={`categories:'${algoliaCategoryName}'`} />
+            <Configure
+              filters={`categories:'${escapeApostrophes(algoliaCategoryName)}'`}
```

Comment:
Suggestion: lift this data manipulation to the `set` function on the _useState_ hook so you situate state management in one place. Line https://github.com/Exygy/askdarcel-web/pull/91/files#diff-966ba7edc2fb21d0d2d4a5ebdd645fca036239ebe2c557e218eb3e6aa70dd973R87 would become: 
```
setParentCategory(escapeApostrophes(serviceCategory));
```
Then the components that use this data can trust the data is correct no matter where they get it from.